src/index.ts

```diff
@@ -19,6 +20,9 @@ export function createLanguageService(
 	setPreferences?(preferences: ts.UserPreferences): void;
 	projectUpdated?(updatedProjectDirectory: string): void;
 } {
+	if (semver.gte(ts.version, '5.6.1')) {
```

Comment:
Double check, was the change introduced in 5.6.1 and not 5.6.0? If yes, can you add a comment?

Reply:
Yes. it is 5.6.1 just checked it is actually 5.6.1-rc. They changed the tag naming in 5.6 😅.